### PR TITLE
Updated cob4 footprint and added missing slash

### PR DESCRIPTION
**Pull request by NadiaHG**

### CODE REVIEW

File: cob_linear_nav/src/cob_linear_nav.cpp [hunk 1]
```diff
@@ -386,9 +386,11 @@ bool NodeClass::getUseMoveAction(void)
 
 geometry_msgs::PoseStamped NodeClass::transformGoalToMap(geometry_msgs::PoseStamped goal_pose) {
   geometry_msgs::PoseStamped goal_global_;
-  if(goal_pose.header.frame_id == global_frame_) return goal_pose;
-  else if(tf_listener_.canTransform(global_frame_, goal_pose.header.frame_id, ros::Time(0), new std::string)) {
-    tf_listener_.transformPose(global_frame_, ros::Time(0), goal_pose, "base_link", goal_global_);
+  if((goal_pose.header.frame_id == global_frame_) or (goal_pose.header.frame_id == "/" + global_frame_)){
```
Comment (mgruhler) [inline cob_linear_nav/src/cob_linear_nav.cpp:389]: Why do you want to prefix the frame_id with a slash? As far as I know, in tf2 the slashes will get stripped anyway...

File: cob_linear_nav/src/cob_linear_nav.cpp [hunk 2]
```diff
@@ -386,9 +386,11 @@ bool NodeClass::getUseMoveAction(void)
...
+    return goal_pose;
+  }
+  else if(tf_listener_.canTransform(global_frame_, goal_pose.header.frame_id, ros::Time::now()-ros::Duration(1), new std::string)) {
+    tf_listener_.transformPose(global_frame_, ros::Time::now()-ros::Duration(1), goal_pose, "base_link", goal_global_);
```
Comment (mgruhler) [inline cob_linear_nav/src/cob_linear_nav.cpp:393]: Why do you replace the timestamps like this? The old one took the latest available transform, the new one will take the  transform from a second ago...

**Comment by floweisshardt** [comment]
@ipa-nhg: both (slash and footprint) are temporary fixes to move cob4-2 base, but that's nothing I'd like to commit to the ipa320 version. Therefore we should cleanup and debug linear_nav further and choose a reasonable footprint for linear_nav as well as for dwa and tr.
@ipa-mig: please do not merge

**Comment by mgruhler** [comment]
@ipa-fmw @ipa-nhg Alright. I'll close it.

**Pull request closed without merging**